packages/react-components/react-combobox/.babelrc.json

```diff
@@ -1,4 +1,4 @@
 {
-  "presets": ["@griffel"],
+  "extends": "../../../.babelrc-v9.json",
   "plugins": ["annotate-pure-calls", "@babel/transform-react-pure-annotations"]
```

Comment:
Wondering why these plugins are not defined in `.babelrc-v9.json` itself. Are these reasons for that?

Reply:
No major reason - majority of the packages required both those plugins but a few only required `annotate-pure-calls`. So if plugins were also part of the root `babelrc-v9.json` file, the only plugin that would be present there is `annotate-pure-calls` while still requiring each individual v9 package to manually add the `@babel/transform-react-pure-annotations` plugin which didn't seem like a necessary abstraction. It's definitely a cleanup that I can do in a separate PR though!